src/bin/find.js

```diff
@@ -11,58 +10,59 @@ var options = {
   error: ['error'],
   nc: [],
   core: ['core'],
-  verbose: ['verbose', 'v'],
-}
+  verbose: ['verbose', 'v']
+};
 
-var argv = require('yargs')
+const argv = require('yargs')
   .boolean(Object.keys(options))
   .alias(options)
   .default('error', true)
   .default('core', true)
-  .argv
+  .argv;
 
-var getRuleURI = require('eslint-rule-documentation')
+const getRuleURI = require('eslint-rule-documentation');
+const getRuleFinder = require('../lib/rule-finder');
+const cli = require('../lib/cli-util');
 
-var cli = require('../lib/cli-util')
+let processExitCode = argv.u && (!argv.n || argv.error) ? 1 : 0;
```

Comment:
Can be ignored, I got it: it _may_ be reassigned on [line 59](https://github.com/sarbbottam/eslint-find-rules/pull/200/files/8eefc9a87197c4f0daf37eb506b9e37b9e8ff00b#diff-b2f762d6c5e0df4a7bbbca595b7b0678R61).